src/register/register-gnome/gnucash-sheet-private.c

```diff
@@ -423,42 +423,41 @@ draw_cell (GnucashSheet *sheet,
     /* top */
     draw_cell_line (cr, bg_color,
                     (borders.top >= borders.left ? x : x + 1.0),
-                    y + 0.5,
+                    (y == 0 ? y + 0.5 : y - 0.5),
                     (borders.top >= borders.right ?
                      x + width : x + width - 1),
-                    y + 0.5,
+                    (y == 0 ? y + 0.5 : y - 0.5),
                     borders.top);
 
     /* bottom */
     draw_cell_line (cr, bg_color,
-                    (borders.bottom >= borders.left ? x : x + 1),
-                    y + height + 0.5,
+                    (borders.bottom >= borders.left ? x : x + 1.0),
+                    y + height - 0.5,
                     (borders.bottom >= borders.right ?
                      x + width : x + width - 1),
-                    y + height + 0.5,
+                    y + height - 0.5,
                     borders.bottom);
 
     /* left */
     draw_cell_line (cr, bg_color,
-                    x + 0.5,
-                    (borders.left > borders.top ? y : y + 1),
-                    x + 0.5,
+                    (x == 0 ? x + 0.5 : x - 0.5),
+                    (borders.left > borders.top ? y : y),
+                    (x == 0 ? x + 0.5 : x - 0.5),
                     (borders.left > borders.bottom ?
-                     y + height : y + height - 1),
+                     y + height : y + height),
                     borders.left);
```

Comment:
Sorry, I just thought it was a mistake as I could see all these one pixel dots every where.